Mathlib/MeasureTheory/Function/L1Space/HasFiniteIntegral.lean

```diff
@@ -253,6 +299,10 @@ theorem all_ae_tendsto_ofReal_norm (h : ∀ᵐ a ∂μ, Tendsto (fun n => F n a)
     ∀ᵐ a ∂μ, Tendsto (fun n => ENNReal.ofReal ‖F n a‖) atTop <| 𝓝 <| ENNReal.ofReal ‖f a‖ :=
   h.mono fun _ h => tendsto_ofReal <| Tendsto.comp (Continuous.tendsto continuous_norm _) h
 
+theorem all_ae_tendsto_ofReal_norm' (h : ∀ᵐ a ∂μ, Tendsto (fun n ↦ F' n a) atTop <| 𝓝 <| f' a) :
```

Comment:
```suggestion
theorem ae_tendsto_enorm (h : ∀ᵐ a ∂μ, Tendsto (fun n ↦ F' n a) atTop <| 𝓝 <| f' a) :
```
seems more reasonable, right? The naming of the previous lemmas looks also very weird, but it's probably not the role of this PR to fix these.

Reply:
I filed #24354 with some ideas for better names (only the last commit is new).